tests/common/test_cgroupapi.py

```diff
@@ -18,30 +18,37 @@
 from __future__ import print_function
 
 import subprocess
-
-from azurelinuxagent.common.cgroupapi import CGroupsApi, FileSystemCgroupsApi, SystemdCgroupsApi, VM_AGENT_CGROUP_NAME
-from azurelinuxagent.common.exception import ExtensionError, ExtensionErrorCodes
+from azurelinuxagent.common.cgroupapi import CGroupsApi, FileSystemCgroupsApi, SystemdCgroupsApi, CGROUPS_FILE_SYSTEM_ROOT, VM_AGENT_CGROUP_NAME
+from azurelinuxagent.common.exception import CGroupsException, ExtensionError, ExtensionErrorCodes
 from azurelinuxagent.common.future import ustr
 from azurelinuxagent.common.utils import shellutil
 from nose.plugins.attrib import attr
+from tests.utils.cgroups_tools import CGroupsTools
 from tests.tools import *
 
 
-class CGroupsApiTestCase(AgentTestCase):
-    def setUp(self):
+class _MockedFileSystemTestCase(AgentTestCase):
+    def setUp(self, mock_cgroups_file_system_root=True):
```

Comment:
'mock_cgroups_file_system_root' is a left over from a previous iteration... will remove it now